text-adventure-game/text-adventure-game-imp/src/main/java/org/textadventure/game/impl/GameItem.java

```diff
@@ -0,0 +1,32 @@
+package org.textadventure.game.impl;
+
+import org.textadventure.questeditor.api.Action;
+import org.textadventure.questeditor.api.Item;
+
+import java.util.Iterator;
+
+/**
+ * Created by Alexander on 25.12.2015.
+ */
+public class GameItem implements Item {
```

Comment:
Nothing should not inherit from persistent classes. 
You can use the principle of composition instead its